src/routes/console/project-[project]/functions/function-[function]/deployment-[deployment]/+page.ts

```diff
@@ -2,10 +2,17 @@ import { sdk } from '$lib/stores/sdk';
 import { Dependencies } from '$lib/constants';
 import type { PageLoad } from './$types';
 
-export const load: PageLoad = async ({ params, depends }) => {
+export const load: PageLoad = async ({ params, depends, parent }) => {
+    const data = await parent();
     depends(Dependencies.DEPLOYMENT);
 
+    const deployment = await sdk.forProject.functions.getDeployment(
+        params.function,
+        params.deployment
+    );
+
     return {
-        deployment: await sdk.forProject.functions.getDeployment(params.function, params.deployment)
+        deployment,
+        activeDeployment: data.function.deployment === params.deployment ? true : false
```

Comment:
I don't think we need the ternary operator `data.function.deployment === params.deployment` is already either true or false

Reply:
fixed